### PR TITLE
Feature/metadata upload

### PR DESCRIPTION
**Pull request by loremattei**
This PR is aimed to simplify the process of:
- uploading the metadata strings that have to be translated to our translation system 
- downloading translation from the system and update the app store.

_Note_: There are a lot of files added by this PR, but the ones into the `metadata` folder are just text files with the current metadata. 

#### Proposed flow
1. At _code freeze_, update the txt files in the `WordPress/metadata` folder. Then run`fastlane update_ps_strings version:<version>` script that will update the .po file as required. Commit and, at the right moment, merge to `master`.
2. The translation system is already set up to pick the data in the .po file from the `master` branch, so the new/updated strings will be picked and proposed for translation.
3. at _release_, running `fastlane download_metadata_string build_number:<build_number> version:<version>` will download all the available translations from the system and it will update the local files.
4. when the local files are updated, run ` fastlane supply --skip_upload_screenshots` to upload the metadata to the store.

### CODE REVIEW

File: fastlane/metadata/android/de-DE/changelogs/556.txt [hunk 1]
```diff
@@ -0,0 +1,7 @@
+9.9:
```
[the remark this answers is not in the document]
Comment (loremattei) [reply fastlane/metadata/android/de-DE/changelogs/556.txt]: The reason is that the system deletes from the server the entries that it doesn't find anymore in the local filesystem. Downloading all the resources is a quite slow process, so I think having a baseline in the repository is useful (btw: we have this problem on other reps too).

But, maybe we don't need to keep all the changelogs for old version. We would be fine with just the current one.
Does this make sense to you?

[the remark this answers is not in the document]
Comment (mzorz) [reply fastlane/metadata/android/de-DE/changelogs/556.txt]: > the system deletes from the server the entries that it doesn't find anymore in the local filesystem.

Oh I see! Thanks for clarifying that 🙇 

> maybe we don't need to keep all the changelogs for old version. We would be fine with just the current one.

ah that sounds good! and we'd just keep the latest ones, as we can always look back in history with the repo history itself, right? Then that sounds good 👍

**Comment by loremattei** [comment]
Removed all the old changelogs and updated the script to clean the folder at launch

**Comment by mzorz** [comment]
LGTM :shipit:

**Pull request merged**